src/Adapters/AzureManagementAdapter.ts

```diff
@@ -685,7 +685,10 @@ export default class AzureManagementAdapter implements IAzureManagementAdapter {
 
         if (useCache) {
             return this.timeSeriesConnectionCache.getEntity(
-                'adt_adx_connection_information',
+                `${
```

Comment:
should we have a "BuildCacheKey" function so we use the same logci in the set as we do in the get?

Reply:
is this something you would think of @msnyder-msft ? https://github.com/microsoft/iot-cardboard-js/pull/711/commits/30c57457f1884c3f98989b71400cdec3354f8c27